infra/ami.json

```diff
@@ -0,0 +1,49 @@
+{
+  "description": "Packer template for tasking-manager host instance.",
+  "_readme": [
+      "Operating system: Ubuntu 16.04",
+      "Region: us-east-1",
+      "Instance Type: t2.micro",
+      "TODO: Move to Debian Stretch / Buster",
+      "TODO: Switch to NodeJS v10"
+  ],
+  "variables": {
+    "aws_access_key": "{{env `AWS_ACCESS_KEY_ID`}}",
+    "aws_secret_key": "{{env `AWS_SECRET_ACCESS_KEY`}}",
+    "instance_type": "t2.micro",
```

Comment:
Not sure whether this is relevant here, recently we have been using bigger machines.

Reply:
This is simply the machine type it launches to configure in order to make an image out of. The type of instance at launch time is decided by us.